pywbem/cim_operations.py

```diff
@@ -1952,33 +1952,31 @@ def _get_rslt_params(result, namespace):
 
         for p in result:
             if p[0] == 'EndOfSequence':
-                if p[2] is None:
-                    valid_result = True
-                elif isinstance(p[2], six.string_types) and \
+                if isinstance(p[2], six.string_types) and \
                     p[2].lower() in ['true', 'false']:  # noqa: E125
-                    valid_result = True if p[2].lower() == 'true' else False
-                    end_of_sequence = valid_result
+                    end_of_sequence = True if p[2].lower() == 'true' else False
+                    if end_of_sequence is True:
+                        valid_result = True
```

Comment:
This logic renders EndOfSequence=FALSE an invalid result. I don't think that is what we want, particularly because we now require both parameters in the result.